### ISSUE TITLE
Navigating between notes without leaving notes buffer

### ISSUE THREAD

**Issue by joncol** [org-noter/org-noter#69]
Hi, thanks for taking over the project!

Is it possible to configure `org-noter` to navigate between notes *without* the focus being moved to the PDF frame? It would be nice if it was possible (with configuration) that both moving to next/previous note, and `org-noter-sync-current-note` would not move focus away from the notes document.

**Comment by dmitrym0** [comment]
Is #65 similar to what you're asking?

**Comment by petermao** [comment]
No, this is not the same.  Here, I believe the OP wants focus to stay in the same window (doc or notes) where the command is issued.  I do think that it would be **less surprising** for the doc and notes to move with a next/previous command but for the focus not to shift to the document all the time.

For example, I find the current behavior a bit irritating when I need to tweak the position of a precise note manually, which usually takes a few iterations.  Having the focus remain in the same buffer would reduce that by one keychord per cycle.

**Comment by joncol** [comment]
Yes, what @petermao said.

**Comment by petermao** [comment]
Featurewise, PR #71 looks reasonable, but there's a bug somewhere in there (or in emacs itself).  see https://github.com/org-noter/org-noter/pull/71#issuecomment-1899906451.

I don't have time right now to chase this problem down, but if some enterprising user wants to take it on, I'll take the time to vet the changes.
